account_multicurrency_revaluation/wizard/wizard_currency_revaluation.py

```diff
@@ -56,7 +56,7 @@ def _get_default_revaluation_account_ids(self):
         comodel_name="account.journal",
         string="Journal",
         domain=[("type", "=", "general")],
-        help="You can set the default journal in company settings.",
+        help="You can set the default journal in company settings. ",
```

Comment:
This one only has one line so no need space.
```suggestion
        help="You can set the default journal in company settings.",
```